picard/ui/edittagdialog.py

```diff
@@ -22,24 +23,55 @@
     QtWidgets,
 )
 
+from picard.const import (
+    RELEASE_COUNTRIES,
+    RELEASE_FORMATS,
+    RELEASE_PRIMARY_GROUPS,
+    RELEASE_SECONDARY_GROUPS,
+    RELEASE_STATUS,
+)
 from picard.util.tags import TAG_NAMES
 
 from picard.ui import PicardDialog
 from picard.ui.ui_edittagdialog import Ui_EditTagDialog
 
 
+AUTOCOMPLETE_RELEASE_TYPES = [s.lower() for s
+                              in sorted(list(RELEASE_PRIMARY_GROUPS.keys())) + sorted(list(RELEASE_SECONDARY_GROUPS.keys()))]
+AUTOCOMPLETE_RELEASE_STATUS = sorted([s.lower() for s in RELEASE_STATUS])
+AUTOCOMPLETE_RELEASE_COUNTRIES = sorted(RELEASE_COUNTRIES.keys())
```

Comment:
`sorted()` will cast to iterable, and for dicts that's keys, so explicit call to `keys()` is not needed.